http-servers/src/main/java/com/palantir/remoting/http/server/WebPreconditions.java

```diff
@@ -0,0 +1,114 @@
+/*
+ * Copyright 2015 Palantir Technologies, Inc. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.palantir.remoting.http.server;
+
+import com.google.common.base.Preconditions;
+import com.google.common.base.Strings;
+import javax.annotation.Nullable;
+import javax.ws.rs.BadRequestException;
```

Comment:
I'm tempted to say no because this is in http-servers, which depends on dropwizard-core. I might be missing something though.